### DISCUSSION TITLE
Date picker - date rangepicker - week start of monday.

### DISCUSSION THREAD

**Discussion by peroxovy** [themesberg/flowbite#128] [Q&A]
Hello, I want to ask if there's possibility to change the starting day of week in datepicker from flowbite?
I did several "tests" but any of option don't work. In the documetation I didn't see the option for changing starting day of week to Monday f.e. 

Anyone can help?

**Comment by erenergul** [comment]
new Datepicker(datepickerEl, {
    autohide: true,
    todayBtn: true,
    weekStart: 1,
    daysOfWeekDisabled: [0,1],
    clearBtn: true,
});

Thats how i do that. You can use weekStart:1 to start on Monday.

**Reply by peroxovy** [reply]
Hello, but where do you initialize it? I'm using plain HTML in Laravel Blade.
